tests/unit/scripting.tcl

```diff
@@ -146,6 +146,13 @@ start_server {tags {"scripting"}} {
         } 1 x
     } {number 1}
 
+    test {EVAL - Lua number -> Redis integer conversion} {
+        run_script {
```

Comment:
```suggestion
    r del hash
    test {EVAL - Lua number -> Redis integer conversion} {
        run_script {
```

Reply:
is that needed to make sure the hash is in a known state before the test?